### DISCUSSION TITLE
Invalid FEN renders the page unresponsive.

### DISCUSSION THREAD

**Discussion by drishti307** [ashishpapanai/chessJS#102] [Ideas]
**Description**
Invalid FEN renders the page unresponsive.

**Proposed Solution**
Display a prompt, and return to the game.

**Comment by ashishpapanai** [comment]
#### Where to find the bug: 
`js > board.js > function ParseFen(fen) > case default >` 

_Add a modal or browser alert for invalid FEN and refresh the page to give a fresh copy of the site to the user._

**Comment by ayeshaqaisar** [comment]
Can I work on this issue? SWoC 2021 Participant

**Comment by ashishpapanai** [comment]
Sure. Please DM on slack to tell your approach to this issue.

**Comment by ayeshaqaisar** [comment]
> Sure. Please DM on slack to tell your approach to this issue.
> […](#)
> On Sun, 21 Feb, 2021, 1:06 pm Ayesha Qaisar, ***@***.***> wrote: Can I work on this issue? SWoC 2021 Participant — You are receiving this because you commented. Reply to this email directly, view it on GitHub <[#69 (comment)](https://github.com/ashishpapanai/chessJS/issues/69#issuecomment-782814528)>, or unsubscribe <https://github.com/notifications/unsubscribe-auth/AMNVNZD34IBZ3S7JTWLULWLTACZXZANCNFSM4V55P74A> .

Okay
